Sources/JoyfillComponents/Components/Table/ViewTable.swift

```diff
@@ -1225,6 +1225,23 @@ extension ViewTable: UICollectionViewDelegate, UICollectionViewDataSource, UICol
                 cell.contentView.subviews.forEach { $0.removeFromSuperview() }
                 cell.setupSeparator()
                 cell.setDropdownInTableColumn()
+                
+            } else if tableColumnType[tableIndexNo][indexPath.row] == FieldTypes.image {
+                if indexPath.section > 0 {
+                    setImageValue(cell: cell, indexPath: indexPath)
+                }
+                if cell.imageCountLabel.labelText == "  Dummy Text" {
```

Comment:
Same here why is there Dummy Text here again?

Reply:
This " Dummy Text" here is used to hide or unhide image count label in tableImageColumn. As if any cell doesn't have any image then its label gives this value and on this bases I put condition here.
This is used for internal model (where user navigate after clicking on view button).

Reply:
@nishantKumar9408 / @swutask  I will say this again as it has been a couple times now so please store this info for later on. **"NEVER have dummy text or default values for fields hardcoded"**